rbac/management/tenant_mapping/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""TenantMapping model."""
+
+import logging
+import uuid
+
+from django.db import models
+
+from api.models import Tenant
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class TenantMapping(models.Model):
+    """Tenant mappings to V2 domain concepts."""
+
+    tenant = models.OneToOneField(Tenant, on_delete=models.CASCADE)
+    default_group_uuid = models.UUIDField(default=uuid.uuid4, editable=False, null=False)
+    default_admin_group_uuid = models.UUIDField(default=uuid.uuid4, editable=False, null=False)
+    default_user_role_binding_uuid = models.UUIDField(default=uuid.uuid4, editable=False, null=False)
```

Comment:
How "user" is meant here ? Is it like non-admin ?  If so maybe it could be `default_role_binding_uuid`, as there is in RBAC default_group vs default_admin_group...

Reply:
Good call – fixed